recipes/web3/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "web3" %}
+{% set version = "5.12.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f378a773886122d2126d7ba81e9f5b6d2e7b4def7e88543699bf85a951835942
+
+build:
+  number: 0
+  skip: true  # [py==38]
```

Comment:
Maybe just add a comment to yourself as to why you are skipping python 3.8 referencing some issue you raised on github.